content/updates/2029-02-06-update.md

```diff
@@ -0,0 +1,41 @@
++++
+title = "~2020.2.6 urbit.org update"
+date = 2020-02-06
+description = "A brief update from the Tlon team for ~2020.2.6."
+slug = "/updates/2020-02-06"
+[extra]
+author = "Corrina"
+ship = "roslet-tanner"
+image = "https://media.urbit.org/site/posts/updates/~2020.02.06-update.jpg"
++++
+
+![pajaro dunes](https://media.urbit.org/site/posts/updates/~2020.02.06-update.jpg)
+
+It’s been a busy 2020 already, and we’re excited about everything that’s going live this year. Here’s the latest from the Tlon team:
+
+1. Last month’s updates to Arvo and Bridge included:
+Fixes related to the network stability vote.
+[Updates](https://github.com/urbit/urbit/pull/1996) to both Gall and Ames (our application sandbox and networking module, respectively).
+[Support in Bridge](https://github.com/urbit/bridge/pull/335) for withdrawing stars from the linear release contracts.
+ 
+2. Gavin (`~ridlur-figbud`) wrote about how we created sigils, the visual representation of Urbit IDs. Read the post [here](https://urbit.org/blog/creating-sigils/). 
+
+3. We’re pleased to announce the latest round of our semiannual gifts to Urbit contributors. It’s our way of rewarding the hard work of contributors to the project over the last six months. To see all the contributions we’ve recognized, take a look at the [History](https://grants.urbit.org/history) section on the Grants website.
+    - ~bannum-magtus has been tireless in their efforts as a community Guide in the #ship-starting-support channel on our [Discord server](https://discord.gg/C9ENTt3), and as an Urbit evangelizer.
+    - ~nolmeg-hobdyn could also be found nearly every day volunteering as a Guide in #ship-starting-support.
+    - ~minder-folden was a deep font of Urbit-related content, outreach, and support.
+    - ~risruc-habteb provided invaluable, consistent support in urbit-help.
+    - ~rabsef-bicrym for their efforts to teach Hoon to the community.
+    - [~dinleb-rambep](https://github.com/pkova) brought a steady stream of useful contributions to the [urbit repository](https://github.com/urbit/urbit/pulls?utf8=%E2%9C%93&q=author%3Apkova). 
+
+4. Looking for a more formal way to contribute and get rewarded? Look for a [Bounty](https://grants.urbit.org/bounties) to claim, or pitch a [Proposal](https://grants.urbit.org/proposals) of your own. We’re always on the lookout for community members who distinguish themselves through their contributions. 
+
+5. Community member Reid is hosting a Dallas-Fort Worth meetup on February 12. [RSVP](https://www.meetup.com/Urbit-DFW/events/268194997/?rv=ea1_v2&_xtd=gatlbWFpbF9jbGlja9oAJGUwMGVjNGVmLTM3YTMtNDI1Yy05MDY1LTMyZGNiZjIxNDc2MA) to join.
+ 
+6. `~rabsef-bicrym` is coordinating a Denver meetup on February 13 during [ETHDenver](https://www.ethdenver.com/). Come say hi to other community members and folks from Tlon. For more details and to RSVP contact `~rabsef-bicrym` at `denver.urbit.meetup@protonmail.com`.
```

Comment:
Writing the email as inline code breaks the layout on mobile (no wrapping behaviour because it’s intended for inline code). Consider just making it a link, like `[this address]`.

Reply:
Done!